README.md

```diff
@@ -87,91 +108,80 @@ echo -e "admin\n${CONCOURSE_ATC_PASSWORD}" | \
    ${FLY_CMD} -t ${DEPLOY_ENV} login -k -c "https://${DEPLOY_ENV}-concourse.cf.paas.alphagov.co.uk"
 ```
 
-## BOSH and Cloud Foundry
+### Destroy
 
-### Microbosh deployment from concourse bootstrap
+Run the `destroy-deployer` pipeline from your *Bootstrap Concourse*.
 
-These pipelines will deploy/destroy a microbosh using bosh-init. It will use the IAM profile `deployer-concourse` that was given to the Concourse instance.
+## MicroBOSH
 
-#### Prerequisites
+MicroBOSH is responsible for deploying [CloudFoundry](#cloudfoundry) and
+supporting services for the platform.
 
-To run the following steps you must have these prerequisites:
+### Prerequisites
 
- * The `create-deployer` pipeline has been run from a bootstrap concourse.
- * A VPC is setup and there is a bucket for the state files.
- * The `fly` command is installed and available
- * A running concourse configured in the CLI, with target name `$DEPLOY_ENV`
+You will need a working [Deployer Concourse](#deployer-concourse).
 
-#### Pipelines: create-microbosh and destroy-microbosh
+### Deploy
 
-To setup the pipeline `create-microbosh`:
+Run the `create-microbosh` pipeline which can be setup by running:
```

Comment:
It would look more correct with "set up" when using the verb ("setup" for a noun). Ref: http://www.future-perfect.co.uk/grammar-tip/is-it-setup-set-up-or-set-up/